src/styles/item/_sidebar.scss

```
@@ -36,13 +36,15 @@ section.sidebar {
     .feat-details {
         display: flex;
         flex-flow: column nowrap;
-        gap: 0.25rem;
+        gap: var(--space-4);
         overflow: hidden scroll;
-        padding: 0 0.25rem 0 0;
+        padding: 0 var(--space-4) 0 0;
```

```suggestion
        padding: var(--space-8) var(--space-4) 0 0;
```

Makes the alignment for the sidebar a bit prettier. I was gonna do this anyways, but doing it here avoids the conflict :)